sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -785,6 +796,39 @@ def list_blobs(self, name_starts_with=None, include=None, **kwargs):
             command, prefix=name_starts_with, results_per_page=results_per_page,
             page_iterator_class=BlobPropertiesPaged)
 
+    @distributed_trace
+    def list_blob_names(self, **kwargs: Any) -> ItemPaged[str]:
+        """Returns a generator to list the names of blobs under the specified container.
+        The generator will lazily follow the continuation tokens returned by
+        the service.
+
+        :keyword str name_starts_with:
```

Comment:
Do we need to support the `include` keyword param?

Reply:
As discussed offline, will leave this out for now.